src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -89,16 +94,6 @@ public override void _Ready()
         }
     }
 
-    public override void _Process(float delta)
```

Comment:
Please keep this approach where the data is only rebuilt *once* per any sequence of changes to the data on the next frame, instead of setting one property causing the expensive rebuild to happen. Or do you have a very good reason why this must be changed?